spec/session_spec.coffee

```diff
@@ -508,6 +509,32 @@ describe 'Session', ->
 
             expect(@biskoto.get(@basic_meta_cookie_name)).to.deep.equal metadata
 
+        context 'and shop_cookie_policy is specified as basic', ->
+          beforeEach ->
+            @shop_cookie_policy = 'basic'
```

Comment:
This line does not effect the run of the spec because init is called a little bit differently